### PR TITLE
10 category card

### PR DESCRIPTION
**Pull request by LaidBengli**

### CODE REVIEW

File: src/components/categorycard/CategoryCard.jsx [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import React from "react";
```
Comment (darinetag) [inline src/components/categorycard/CategoryCard.jsx]: for cleaner code, remove the import from react

File: src/components/categorycard/CategoryCard.jsx [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+const CategoryCard = ({ category }) => {
+    return (
+        <div className='max-w-sm mx-auto bg-white rounded-xl shadow-md overflow-hidden'>
+            <img
```
Comment (darinetag) [inline src/components/categorycard/CategoryCard.jsx:4]: nicely done

**Review by darinetag — APPROVED** [review]

**Comment by darinetag** [comment]
@ismail-benlaredj can you check the prettier issue then close the PR.

**Review by SaraBegache — CHANGES_REQUESTED** [review]
Yes please apply what @darinetag suggested, thank you.

**Review by ismail-benlaredj — APPROVED** [review]

**Pull request merged**